src/resmom/mom_main.c

```diff
@@ -8447,19 +8447,19 @@ main(int argc, char *argv[])
 				rlimit.rlim_cur = MIN_STACK_LIMIT;
 				rlimit.rlim_max = MIN_STACK_LIMIT;
 				if (setrlimit64(RLIMIT_STACK, &rlimit) == -1) {
+					char msgbuf[] = "Stack limit setting failed";
 					curerror = errno;
-					sprintf(log_buffer, "Stack limit setting failed");
-					log_err(curerror, __func__, log_buffer);
-					sprintf(log_buffer, "%s errno=%d", log_buffer, curerror);
+					log_err(curerror, __func__, msgbuf);
+					sprintf(log_buffer, "%s errno=%d", msgbuf, curerror);
 					log_record(PBSEVENT_ERROR, PBS_EVENTCLASS_SERVER, LOG_ERR, (char *)__func__, log_buffer);
 					exit(1);
 				}
 			}
 		} else {
+			char msgbuf[] = "Getting current Stack limit failed";
 			curerror = errno;
-			sprintf(log_buffer, "Getting current Stack limit failed");
-			log_err(curerror, __func__, log_buffer);
-			sprintf(log_buffer, "%s errno=%d", log_buffer, curerror);
+			log_err(curerror, __func__, msgbuf);
+			sprintf(log_buffer, "%s errno=%d", msgbuf, curerror);
```

Comment:
Doesn't have to be since we know the length of the string we are printing plus the length of an integer could not possibly exceed the log buffer size. I updated it for the sake of consistency.